screens/Profile/MyProfileScreen/styles.ts

```diff
@@ -1,13 +1,57 @@
-import { StyleSheet } from 'react-native'
+import { StyleSheet } from "react-native"
+import { black, lightPeach, peach } from "../../../assets/colors/colors"
 
 export const styles = StyleSheet.create({
+  button: {
+    alignItems: "center",
+    backgroundColor: lightPeach,
+    borderColor: peach,
+    borderRadius: 10,
+    borderWidth: 2,
+    height: 38,
+    justifyContent: "center",
+    marginLeft: 14,
+    width: 90,
+  },
+  buttonText: {
+    alignItems: "center",
+    fontSize: 12,
+    fontWeight: "bold",
```

Comment:
I can also see that you are not using our custom font. When you start using it, you will need to use different `fontFamily` instead of `fontWeight`